packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md

```diff
@@ -1008,4 +1010,28 @@ is not viable in some productive setups.
 | 2.84.0 | `false` | `true` |
 
 
+### @aws-cdk/aws-autoscaling:disableDefaultLaunchConfigCreation
```

Comment:
Since we are now creating a default launch template I think we should change this name since it implies that when enabled we won't create anything.

Maybe `defaultLaunchTemplateInsteadOfLaunchConfig`. It's kind of verbose, so if you have anything better...